src/main/java/microsoft/exchange/webservices/data/EwsXmlReader.java

```diff
@@ -422,7 +422,7 @@ public String readValue() throws XMLStreamException,
     } else if (this.presentEvent.getEventType() == XmlNodeType.CHARACTERS
         && this.presentEvent.isCharacters()) {
                         /*
-			 * if(this.presentEvent.asCharacters().getData().equals("<")) {
```

Comment:
@barisaydinoglu I also noticed this kind of gap in the styleguide and also realized that the provided formatters of the google-site have it´s differences. Running these against checkstyle also shows that there is something wrong with it and it is just bad that this topic keeps chasing us. Perhabs we can have a seperate PR with the things you noticed changed in the codeStyleSettings.xml. I opened #158 for this. Nevertheless the files with untouched code should be reverted in my opinion.